src/classes/Pollers/Server.php

```diff
@@ -245,7 +245,7 @@ private function getProbeResult(string $base_url): array
 		$handle = curl_init();
 		$options = array(
 			//Timeouts
-			CURLOPT_TIMEOUT => max($this->settings['probe_timeout'], 1), //Minimum of 1 second timeout.
+			CURLOPT_TIMEOUT => max($this->settings['probe_timeout'], 3), //Minimum of 3 second timeout.
```

Comment:
1 second is the absolute minimum but you can increase the timeout itself in your local settings file. Please remove this change from your PR.

Reply:
Thanks for the hint @MrPetovan !

Could you tell us how this configuration looks like in practice?

Reply:
Sure, here's a sample `config/local.json` file with the setting updated:
```json
{
    "database": {
        "driver": "mysql",
        "hostname": "localhost",
        "database": "friendica-directory",
        "username": "friendica-directory",
        "password": "🤡"
    },
    "poller": {
        "server_poll_delay": 28800,
        "profile_poll_delay": 36000,
        "probe_timeout": 10
    },
    "logger": {
        "level": 250
    }
}
```

Reply:
Great! Thank very much, @MrPetovan